app/backend/src/couchers/servicers/admin.py

```diff
@@ -288,6 +293,87 @@ def CreateCommunity(self, request, context, session):
 
         return community_to_pb(session, node, context)
 
+    def UpdateCommunity(self, request, context, session):
+        cluster = session.execute(
+            select(Cluster).where(Cluster.id == request.community_id).where(Cluster.deleted == None)
+        ).scalar_one_or_none()
+        if not cluster:
+            context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+
+        node = session.execute(select(Node).where(Node.id == cluster.parent_node_id)).scalar_one_or_none()
+        if not node:
+            context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+
+        name = request.name if request.name else cluster.name
+        description = request.description if request.description else cluster.description
+
+        geom = node.geom
+        if request.geojson:
+            geom = shape(json.loads(request.geojson))
+            if geom.geom_type != "MultiPolygon":
+                context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.NO_MULTIPOLYGON)
+            geom = from_shape(geom)
+
+        if request.parent_node_id != 0 and request.parent_node_id != node.parent_node_id:
+            parent_node_id = request.parent_node_id
+
+            new_node = Node(geom=geom, parent_node_id=parent_node_id)
+            session.add(new_node)
+
+            now = datetime.utcnow()
```

Comment:
Please use `now()` from `utils.py` (to be consistent, also I'm not sure if `utcnow` (which is naive, I believe) works correctly if the server TZ is not UTC)